--- ORBit2-2.3.104/src/orb/dynamic/Makefile.am~	Wed Feb  6 06:14:58 2002
+++ ORBit2-2.3.104/src/orb/dynamic/Makefile.am	Wed Feb  6 06:14:58 2002
@@ -15,8 +15,6 @@
 	$(ORBIT_DEBUG_CFLAGS)				\
 	$(ORBIT_CFLAGS)
 
-DYNAMIC_IDLOUT=dynamic-defs-common.c
-
 IDL_FLAGS= -I$(top_srcdir)/src/idl/CORBA_PIDL			\
 	--define=Object=OObject --define=TypeCode=TTypeCode	\
 	-I$(top_srcdir)/src/idl/CORBA				\
--- ORBit2-2.3.104/src/orb/orb-core/Makefile.am~	Wed Feb  6 06:24:16 2002
+++ ORBit2-2.3.104/src/orb/orb-core/Makefile.am	Wed Feb  6 06:24:16 2002
@@ -49,7 +49,6 @@
 IFACE_IDLOUT=orbit-interface-common.c
 
 $(liborb_core_la_OBJECTS) : $(BUILT_SOURCES)
-IDL_COMPILER = $(top_builddir)/src/idl-compiler/orbit-idl-2
 
 IDL_FLAGS = -I$(top_srcdir)/src/idl/CORBA_PIDL			\
 	-I$(top_srcdir)/src/idl/CORBA				\
--- ORBit2-2.3.104/Makefile.am~	Wed Feb  6 06:25:12 2002
+++ ORBit2-2.3.104/Makefile.am	Wed Feb  6 06:25:12 2002
@@ -8,7 +8,8 @@
 	ORBit2.m4              \
 	ORBit.spec             \
 	HACKING                \
-	MAINTAINERS
+	MAINTAINERS            \
+	Makefile.shared
 
 rpms: distcheck
 	rpm -ta $(top_builddir)/@PACKAGE@-@VERSION@.tar.gz
