diff -Nur ORBit2-2.4.1/test/everything/Makefile.am ORBit2-2.4.1.new/test/everything/Makefile.am
--- ORBit2-2.4.1/test/everything/Makefile.am	Thu Jun 13 14:49:55 2002
+++ ORBit2-2.4.1.new/test/everything/Makefile.am	Thu Aug 15 00:47:34 2002
@@ -33,7 +33,7 @@
 client_SOURCES=${EVERYTHING_BUILT} client.c everything.idl constants.h
 client_DEPENDENCIES=${EVERYTHING_BUILT} $(included_src)
 client_LDADD = $(LDADD)
-client_LDFLAGS = -static -module
+client_LDFLAGS = -dynamic -module
 
 included_src = \
 	anyServer.c	 \
@@ -56,7 +56,7 @@
 Everything_module_la_LIBADD = \
         $(top_builddir)/src/orb/libORBit-2.la
 server_SOURCES=server.c ${EVERYTHING_BUILT} everything.idl constants.h
-server_LDFLAGS = -static -module
+server_LDFLAGS = -dynamic -module
 server.c : $(included_src)
 	touch server.c
 server_DEPENDENCIES=${EVERYTHING_BUILT} $(included_src)
@@ -65,13 +65,10 @@
 IDL_FILES= everything.idl
 include $(top_srcdir)/Makefile.shared
 
-BUILT_SOURCES = ${EVERYTHING_BUILT} everything-imodule.c
-CLEANFILES = ${BUILT_SOURCES} iorfile
-
 TESTS = $(srcdir)/test.sh
 
 BUILT_SOURCES = $(EVERYTHING_IDLOUT)
-CLEANFILES = $(BUILT_SOURCES)
+CLEANFILES = $(BUILT_SOURCES) iorfile
 EXTRA_DIST = $(IDL_FILES) test.sh $(included_src)
 
 dist-hook:
