--- linc2/src/linc.c	14 Apr 2005 06:21:54 -0000
+++ linc2/src/linc.c	10 May 2005 10:51:52 -0000
@@ -456,7 +456,7 @@ link_exec_set_io_thread (gpointer data, 
 		 link_mainloop_handle_input, NULL);
 	
 	link_io_thread = g_thread_create_full
-		(link_io_thread_fn, NULL, 0, TRUE, FALSE,
+		(link_io_thread_fn, NULL, 256 * 1024, TRUE, FALSE,
 		 G_THREAD_PRIORITY_NORMAL, &error);
 	
 	if (!link_io_thread || error)
