diff -Nur ORBit2-2.3.102/Makefile.in ORBit2-2.3.102.new/Makefile.in
--- ORBit2-2.3.102/Makefile.in	Wed Jan  9 16:23:57 2002
+++ ORBit2-2.3.102.new/Makefile.in	Wed Jan 16 00:24:00 2002
@@ -96,7 +96,7 @@
 
 AUTOMAKE_OPTIONS = 1.3
 
-SUBDIRS = src include test docs
+SUBDIRS = src include docs
 
 bin_SCRIPTS = orbit2-config
 

diff -Nur ORBit2-2.3.102/configure.in ORBit2-2.3.102.new/configure.in
--- ORBit2-2.3.102/configure.in	Wed Jan  9 16:21:22 2002
+++ ORBit2-2.3.102.new/configure.in	Tue Jan 15 01:45:11 2002
@@ -219,10 +219,6 @@
 include/orbit/orb-core/Makefile
 include/orbit/poa/Makefile
 include/orbit/dynamic/Makefile
-test/Makefile
-test/everything/Makefile
-test/inhibit/Makefile
-test/poa/Makefile
 docs/Makefile
 docs/devel/Makefile
 docs/internals/Makefile
