--- ORBit2-2.3.109/test/poa/Makefile.am~	Fri May 17 09:30:47 2002
+++ ORBit2-2.3.109/test/poa/Makefile.am	Fri May 17 09:30:47 2002
@@ -14,7 +14,7 @@
                  poatest-basic09.c \
                  poatest-basic10.c
 
-#                 poatest-basic05.c \
+#                 poatest-basic05.c
 
 TESTS = poatest-basic01 \
         poatest-basic02 \
@@ -26,7 +26,7 @@
         poatest-basic09 \
         poatest-basic10
 
-#        poatest-basic05 \
+#        poatest-basic05
 
 check_PROGRAMS = $(TESTS)
 
