diff -urN ORBit2-2.7.1.orig/src/orb/dynamic/Makefile.am ORBit2-2.7.1/src/orb/dynamic/Makefile.am
--- ORBit2-2.7.1.orig/src/orb/dynamic/Makefile.am	Sat May  3 12:14:43 2003
+++ ORBit2-2.7.1/src/orb/dynamic/Makefile.am	Sat May  3 12:15:24 2003
@@ -16,8 +16,6 @@
 	$(DISABLE_DEPRECATED_CFLAGS)			\
 	$(ORBIT_CFLAGS)
 
-DYNAMIC_IDLOUT=dynamic-defs-common.c
-
 IDL_FLAGS= -I$(top_srcdir)/src/idl/CORBA_PIDL			\
 	--define=Object=OObject --define=TypeCode=TTypeCode	\
 	-I$(top_srcdir)/src/idl/CORBA				\
diff -urN ORBit2-2.7.1.orig/src/orb/orb-core/Makefile.am ORBit2-2.7.1/src/orb/orb-core/Makefile.am
--- ORBit2-2.7.1.orig/src/orb/orb-core/Makefile.am	Sat May  3 12:14:43 2003
+++ ORBit2-2.7.1/src/orb/orb-core/Makefile.am	Sat May  3 12:24:22 2003
@@ -53,7 +53,6 @@
 IFACE_IDLOUT=orbit-interface-common.c
 
 $(liborb_core_la_OBJECTS) : $(BUILT_SOURCES)
-IDL_COMPILER = $(top_builddir)/src/idl-compiler/orbit-idl-2
 
 IDL_FLAGS = -I$(top_srcdir)/src/idl/CORBA_PIDL			\
 	-I$(top_srcdir)/src/idl/CORBA				\
diff -urN ORBit2-2.7.1.orig/test/poa/Makefile.am ORBit2-2.7.1/test/poa/Makefile.am
--- ORBit2-2.7.1.orig/test/poa/Makefile.am	Sat May  3 12:14:43 2003
+++ ORBit2-2.7.1/test/poa/Makefile.am	Sat May  3 12:18:10 2003
@@ -15,7 +15,7 @@
                  poatest-basic10.c \
                  poatest-basic11.c
 
-#                 poatest-basic05.c \
+#                 poatest-basic05.c 
 
 TESTS = poatest-basic01 \
         poatest-basic02 \
